state/protocol/badger/state.go

```diff
@@ -52,8 +65,10 @@ func Bootstrap(
 	}
 	state := newState(metrics, db, headers, seals, results, blocks, setups, commits, statuses)
 
-	if err := isValidRootSnapshot(root); err != nil {
-		return nil, fmt.Errorf("cannot bootstrap invalid root snapshot: %w", err)
+	if !skipValidations {
+		if err := isValidRootSnapshot(root); err != nil {
+			return nil, fmt.Errorf("cannot bootstrap invalid root snapshot: %w", err)
+		}
```

Comment:
Do we really want to skip `isValidRootSnapshot` entirely? Or do we only want to skip certain checks inside that function?

Reply:
so for the unstaked AN, the root snapshot is guaranteed to be correct since we are delivering a signed version of the the file and asking the operator to verify the signature. Hence, it should be safe to ignore the validation. I can tweak it to validate checks that can be validated (ones which don't involve the identities).